rclcpp_examples/src/services/add_two_ints_client.cpp

```diff
@@ -41,7 +45,12 @@ int main(int argc, char ** argv)
 
   // TODO(wjwwood): make it like `client->send_request(node, request)->sum`
   // TODO(wjwwood): consider error condition
-  std::cout << "Result of add_two_ints: " << send_request(node, client, request)->sum << std::endl;
+  auto result = send_request(node, client, request);
+  if (result) {
+    printf("Result of add_two_ints: %d\n", result->sum);
```

Comment:
Ugh. The message specification for the service says that `sum` is an int64. Aren't the warnings usually for unsigned types?

I guess there's one way to find out for sure... http://ci.ros2.org/job/ros2_batch_ci_windows/383/